TidepoolKit/Model/Datum/TAlertDatum.swift

```diff
@@ -25,12 +25,15 @@ public class TAlertDatum: TDatum, Decodable {
         case delayed
         case immediate
         case repeating
+        case nextDate
+        case nextDateRepeating
     }
     
     public var name: String?
     public var priority: Priority?
     public var trigger: Trigger?
     public var triggerDelay: TimeInterval?
+    public var triggerDateMatching: DateComponents?
```

Comment:
I think we're going to need to create a TidepoolKit type that partially mirrors `DateComponents`. The reason for doing this is that the backend will need to be updated to support this new field and data type. However, I don't think it is reasonable to expect that the backend support every known option of `DateComponents` as there are a lot that we won't likely use (e.g. era, yearForWeekOfYear, isLeapMonth, ...). I think we need to explicitly state we support only certain fields and define a type here to capture that.

Reply:
@darinkrauss and I chatted and I've agreed to make a specific type that only supports `days`, `hours`, and `minutes` for both `LoopKit` and `TidepoolKit`.  Stay tuned for the changes...